plugins/Eq/EqSpectrumView.h

```diff
@@ -144,7 +144,7 @@ class EqSpectrumView : public QWidget
 		setFixedSize( 400, 200 );
 		QTimer *timer = new QTimer(this);
 		connect(timer, SIGNAL(timeout()), this, SLOT(update()));
-		timer->start(100);
+		timer->start(10);
```

Comment:
I'm not sure we need a 100fps display. Why not set the interval to 20, which gives us about 50fps, which is pretty smooth.